Update setup clerk guide (#962)
clerk/clerk-docs: docs/quickstarts/setup-clerk.mdx

---
description: Set up a new Clerk account and integrate it into a new application.
---

# Set up your Clerk account

Before you can start integrating Clerk into your application, you need to create a Clerk account and set up a new application in the Clerk Dashboard. This guide will walk you through those steps.

<Callout type="info">

>   Migrating from another platform?
> 

Reviewer: ```suggestion
```

Unnecessary with below suggestion

>   You might want to check out one of the [migration guides](/docs/deployments/migrate-overview) to help you move your data to Clerk.

Reviewer: ```suggestion
  If you're migrating from another platform, see the [migration guides](/docs/deployments/migrate-overview) to learn how to move your data to Clerk.
```

"You might want to check out" seems weirdly informal in docs.

</Callout>

<Steps>

### Sign into Clerk

[Create a Clerk account](https://dashboard.clerk.com/sign-up) or [sign into your Clerk Dashboard](https://dashboard.clerk.com/).

### Create a Clerk application

> If you have just created an account for the first time, you will be taken directly to our interactive authentication setup form.

Reviewer: We should decide if we're going to contract things or not, because it's inconsistent throughout the docs. This sentence says "you will" but the next paragraph says "you'll". I think contractions are better because they sound more natural. Which do you prefer?

```suggestion
If you've just created an account for the first time, you'll be taken directly to the interactive authentication setup form.
```

If you don't like the contractions, we still. need to drop "our" here:

```suggestion
If you have just created an account for the first time, you will be taken directly to the interactive authentication setup form.
```

Author: I try my best to remove contractions, as that's what I learned in scientific writing, but maybe it's different for docs. 
The issue with contractions is it can be harder to translate/understand for non-english speakers.

what do you think, though?

Reviewer: The translation issue is important enough to avoid contractions, but I've never heard of that before. I tried googling around but can't find anything that confirms or disproves this. I've also never had to avoid contractions writing for any other teams, and Google's docs style guide (not that they're the final word on this) [encourages their writers to use contractions](https://developers.google.com/style/contractions), which I think they'd avoid if they were aware of this problem.

Author: yeah if that's how most docs do it, then it's fine. we don't have to worry about them then

Otherwise, you'll be redirected to your [Clerk Dashboard](https://dashboard.clerk.com/). To create a new app, select the **Create application** card to be taken to the interactive authentication setup form.

### Select identifiers and social providers

> Once you are in the interactive authentication setup form, you will be asked to build your authentication flow. Here, Clerk provides various options for setting up your sign-up and sign-in flows. You can choose to use email, phone, or username as [identifiers](/docs/authentication/configuration/sign-up-sign-in-options#identifiers), and you can also enable [social authentication providers](/docs/authentication/social-connections/overview).

Reviewer: ```suggestion
Once you are in the interactive authentication setup form, you will be asked to build your authentication flow. Here, Clerk provides various options for setting up your sign-up and sign-in flows. You can choose to use email, phone, or username as [identifiers](/docs/authentication/configuration/sign-up-sign-in-options#identifiers), and you can enable [social authentication providers](/docs/authentication/social-connections/overview).
```
